docs/eventing/sources/sinkbinding/reference.md

```diff
@@ -0,0 +1,171 @@
+# SinkBinding Reference
+
+This topic provides reference information about the configurable fields for the
+SinkBinding object.
+
+
+## SinkBinding
+
+A `SinkBinding` definition supports the following fields:
+
+| Field | Description | Required or optional |
+|-------|-------------|----------------------|
+| [`apiVersion`][kubernetes-overview] | Specifies the API version, for example `sources.knative.dev/v1`. | Required |
+| [`kind`][kubernetes-overview] | Identifies this resource object as a `SinkBinding` object. | Required |
+| [`metadata`][kubernetes-overview] | Specifies metadata that uniquely identifies the `SinkBinding` object. For example, a `name`. | Required |
+| [`spec`][kubernetes-overview] | Specifies the configuration information for this `SinkBinding` object. | Required |
+| [`spec.sink`](#sink-parameter) | A reference to an object that resolves to a URI to use as the sink. | Required |
+| [`spec.subject`](#subject-parameter) | A reference to the resources for which the "runtime contract" is augmented by Binding implementations. | Required |
+| [`spec.ceOverrides`](#cloudevent-overrides) | Defines overrides to control the output format and modifications to the event sent to the sink. | Optional |
+
```

Comment:
again, extra unnecessary line space - maybe adjust whitespace settings in your text editor?

Reply:
I add extra lines around sections to make it easier for me to read a long topic in a text editor. 

Extra lines have never caused problems in the past, but if they cause problems in the Knative docs let me know and I can add a reminder to remove them when I finish a topic.

Reply:
It's really more for consistency, I would really prefer we don't have trailing whitespace for no reason. We used to have a check on github that would fail a test if we had this, but I guess it was removed.

@omerbensaadon it feels like we should have some guidelines on this to keep the docs consistent, e.g. so we don't have double spaces in some places but not others and folks adding new docs know how we expect them to be formatted. I'll leave it up to ya'll to decide which to use since I've already stated my preference, but I think we should at least be consistent.